MentalHealth/Resources.csv

```diff
@@ -0,0 +1,15 @@
+MATES Peer Mentors,"MATES (Mentor Assistance Through Educational Support) is a student outreach program which seeks to promote personal life balance, self confidence, resiliency and academic support for UW students who are currently having academic, social, physical, emotional, cultural, and / or psychological difficulties.","Needles Hall (new addition), 2nd floor","Needles Hall Expansion Office: Monday to Friday: 9am - 4pm, Health Services Centre Office: Monday, Tuesday, Friday:  9:00am - 4:00pm Wednesday & Thursday:  9:00am - 7:00pm",mates@uwaterloo.ca,519-888-4567 ext. 30342,519-888-4567,http://uwmates.weebly.com/,"on campus, peer"
+Centre for Mental Health Research,"The Centre for Mental Health Research (CMHR) offers accessible, effective mental health services for people of all ages in the Region of Waterloo while facilitating cutting-edge research on the development, persistence and treatment of mental health problems and providing comprehensive clinical training in our state-of-the-art facility.","Room 1401, ground floor, Psychology, Anthropology, Sociology (PAS) Building",,cmhr@uwaterloo.ca,519-888-4567 ext. 33842,519-888-4567,https://uwaterloo.ca/mental-health-research/,on campus 
+Health Services,"Health Services provides primary medical care for all registered University of Waterloo students. Our health professionals work to promote a safe, healthy environment for our students.","Take the bridge to Health Services, located across from the Student Life Centre.","Monday to Friday, 8:30am - 12pm, 1pm - 4:30pm (Walk-Ins)",,"519-888-4096 to schedule an appointment, 1-866-797-0000  for free 24/7 advice from a health professional",1-866-797-0000,https://uwaterloo.ca/health-services/,on campus 
+Cambridge Memorial Hospital,,700 Coronation Blvd. Cambridge,24 by 7,information@cmh.org,519-621-2330,519-621-2330,https://www.cmh.org/,community
+Crisis Service of Waterloo Region,"The Canadian Mental Health Association Waterloo Wellington Dufferin Crisis Services provides prevention, assessment, intervention and resolution services for urgent crisis situations in Waterloo Region.","67 King Street E, Canadian Mental Health Association (CMHA) Waterloo Wellington Dufferin, Kitchener, Ontario",24 by 7,,1-866-531-2600,1-866-531-2600,http://www.mentalhealthhelpline.ca/Directory/Program/17278,community
+Delton Glebe Counselling Centre,The Delton Glebe Counselling Centre is a not-for-profit multi-faith counselling agency committed to strengthening emotional and spiritual well-being. ,"177 Albert Street, Waterloo, Ontario","Monday to Thursday,  9am � 8pm",glebecentre@wlu.ca,519-884-3305,519-884-3305,http://glebecounselling.ca/,community
```

Comment:
There's a glitch with character encoding here (and a few other places). Can you identify what character is supposed to be there?

```�``` (this)